tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -9,10 +9,18 @@
 class aspectlistTest:
 
     def test__init__(self):
-        with pytest.raises(aspectTypeError) as exc:
+        list_of_aspect = aspectlist(['CommitMessage.Shortlog',
+                                     'CommitMessage.Body'])
+        mix_of_aspect = aspectlist(['CommitMessage.Shortlog',
+                                    Metadata.CommitMessage.Body])
+        assert isinstance(list_of_aspect, aspectlist)
+        assert list_of_aspect[0] is Metadata.CommitMessage.Shortlog
+        assert list_of_aspect[1] is Metadata.CommitMessage.Body
+        assert list_of_aspect == mix_of_aspect
+
+        with pytest.raises(LookupError) as exc:
             aspectlist(['String'])
-        exc.match("'String' is not an aspectclass or "
-                  'an instance of an aspectclass')
+        exc.match("no aspect named 'String'")
```

Comment:
exception messages should begin with a capital letter: No aspect named ...

Please raise a bug to fix that.